cert-data.yml

```diff
@@ -64,6 +82,9 @@
 Default/OpenEdX/Certificate: &certificate_defaults
   VERSION: 4_programmatic
   TEMPLATEFILE: False
+  font: 
+    name: "OpenSans"
+    type: "Light" 
```

Comment:
By convention, we don't use quotes in YAML unless it contains special characters (one of the format's benefits). If we do (see line 87/108 below), we should use single quotes. But we don't need them here.